geode-redis/src/main/java/org/apache/geode/redis/internal/RedisStats.java

```diff
@@ -171,13 +180,92 @@ public void endCommand(RedisCommandType command, long start) {
   }
 
   public void addClient() {
+    connectionsReceived.incrementAndGet();
+    connectedClients.incrementAndGet();
     stats.incLong(clientId, 1);
```

Comment:
The `clientID` statistic is actually the same as `connectedClients`. We could make the `getClients()` method public and just use that.

Reply:
can likely be addressed in future changes to functionality

Reply:
will likely be addressed in future work on functionality here